sdk/maps/test-resources.json

```diff
@@ -0,0 +1,89 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "baseName": {
+            "type": "string",
+            "defaultValue": "[resourceGroup().name]",
+            "metadata": {
+                "description": "The base resource name."
+            }
+        },
+        "tenantId": {
+            "type": "string",
+            "defaultValue": "72f988bf-86f1-41af-91ab-2d7cd011db47",
+            "metadata": {
+                "description": "The tenant ID to which the application and resources belong."
+            }
+        },
+        "testApplicationOid": {
+            "type": "string",
+            "metadata": {
+                "description": "The client OID to grant access to test resources."
+            }
+        },
+        "testApplicationId": {
+            "type": "string",
+            "metadata": {
+                "description": "The application client id used to run tests."
+            }
+        },
+        "testApplicationSecret": {
+            "type": "string",
+            "metadata": {
+                "description": "The application client secret used to run tests."
+            }
+        }
+    },
+    "variables": {
+       "roleDefinitionId": "[concat('/subscriptions/', subscription().subscriptionId, '/providers/Microsoft.Authorization/roleDefinitions/8f5e0ce6-4f7b-4dcf-bddf-e6f48634a204')]",
+       "mapsAccountName": "[guid(resourceGroup().id, deployment().name, parameters('baseName'))]"
+    },
+    "resources": [
+        {
+            "type": "Microsoft.Maps/accounts",
+            "apiVersion": "2021-12-01-preview",
+            "name": "[variables('mapsAccountName')]",
+            "location": "eastus",
```

Comment:
Our default location is `westus`, so assuming that's a supported region, it's better to inherit from the group location. Sometimes we need to change location en masse when there are capacity constraints.

```
            "location": "[resourceGroup().location]",
```

If it does need to be overridden, it's better to add a `Location` parameter in the `tests.yaml` instead, as this will call our scripts with the location override, which can also be done locally via `eng/common/TestResources/New-TestResources.ps1 -ServiceDirectory maps -Location eastus`. If it's hardcoded here then we lose that flexibility.

Reply:
Thank you for the suggestion.